src/main/java/core/basesyntax/FileWork.java

```diff
@@ -9,6 +15,33 @@
  */
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        StringBuilder buffer = new StringBuilder();
+        try (BufferedReader bReader =
+                     new BufferedReader(new FileReader(new File(fileName)))) {
+            String line = bReader.readLine();
+            while (line != null) {
+                buffer.append(line);
+                line = bReader.readLine();
+            }
+            if (buffer.toString().isEmpty()) {
+                return new String[0];
+            }
+        } catch (IOException e) {
+            throw new RuntimeException();
+        }
+
+        String[] wordsSeacher = buffer.toString().toLowerCase().split(" ");
+        StringBuilder fishWords = new StringBuilder();
+        for (String i : wordsSeacher) {
```

Comment:
try to create a solution without this loop

Reply:
I couldn't change my solution without this loop
I'm sorry(